html-block-and-inline/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Block and Inline</title>
+  </head>
+  <body>
+    <h1>Blue Americana Snuggie&reg; Blanket</h1>
+    <h2>by SnuggieStore</h2>
+    <div>Price: <del>$19.99</del> $14.99</div>
```

Comment:
divs should be used to divide the page, since this is text, a p tag would be better. It is best to use tags designed for text to hold text.